.circleci/config.yml

```diff
@@ -261,27 +244,30 @@ commands:
               pip3 install wheel setuptools
               pip3 install nvidia-pyindex
               pip3 install tabulate
-              pip3 install tensorrt==<< parameters.trt-version-long >> nvidia-cudnn-cu11==<< parameters.cudnn-version-long >>
+              pip3 install tensorrt==<< parameters.trt-version-long >> nvidia-cudnn-cu12==<< parameters.cudnn-version-long >>
               pip3 install pytest parameterized expecttest nox
 
   install-torch-from-index:
     description: "Install Pytorch"
     parameters:
       torch-build:
         type: string
-        default: "2.1.0.dev20230605+cu118"
+        default: "2.1.0.dev20230619+cu121.with.pypi.cudnn"
```

Comment:
Using `.with.pypi.cudnn` resolves the issue detailed by @HolyWu - the sample notebooks provided here: https://github.com/pytorch/TensorRT/pull/2020#issuecomment-1601313192, were very helpful in debugging this issue.

Reply:
Aye. The culprit is that torch 2.0.1 and 2.1.0.dev are having issue with `cuDNN 8.9.x`. When using `.with.pypi.cudnn` package, it installs `nvidia-cudnn-cu12 8.8.1.3` for you.

Reply:
Thanks for the comment - this makes sense, and I've left the CuDNN version at `8.8`, which allows us to avoid usage of `.with.pypi.cudnn` here.